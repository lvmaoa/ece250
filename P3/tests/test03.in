i goat
i donkey
i cheese
i cat
s goat
s dog
spellcheck goat
spellcheck go
exit
