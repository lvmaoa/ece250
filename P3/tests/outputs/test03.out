success
success
success
success
found goat
not found
correct
goat 
